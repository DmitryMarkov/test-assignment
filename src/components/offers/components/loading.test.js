import React from 'react'
import { render } from '@testing-library/react'
import Loading from './loading'

test('Loading should render properly', () => {
  const { asFragment, getByText } = render(<Loading />)

  expect(getByText(/Loading/)).toBeInTheDocument()
  expect(asFragment()).toMatchSnapshot()
})
