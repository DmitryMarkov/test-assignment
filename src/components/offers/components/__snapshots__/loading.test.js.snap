// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render properly 1`] = `
<DocumentFragment>
  <div
    class="css-1wk4bom"
  >
    Loading...
  </div>
</DocumentFragment>
`;
